frontend/public/components/operator-lifecycle-manager/package-manifest.tsx

```diff
@@ -23,7 +23,7 @@ export const PackageManifestRow: React.SFC<PackageManifestRowProps> = (props) =>
   const {obj, catalogSourceName, catalogSourceNamespace, subscription, defaultNS} = props;
   const ns = getActiveNamespace();
   const channel = !_.isEmpty(obj.status.defaultChannel) ? obj.status.channels.find(ch => ch.name === obj.status.defaultChannel) : obj.status.channels[0];
-  const {displayName, icon = [], version, provider} = channel.currentCSVDesc;
+  const {displayName, icon = [], version, provider} = _.get(channel, 'currentCSVDesc');
```

Comment:
I think this would still be a runtime error if `channel` is undefined since we destructure the result. It doesn't look like `_.get` helps here.

Reply:
Actually, OLM validates that all packages in a `CatalogSource` have at least one channel, so this should never occur.